src/filters/simple-filter-directive.js

```diff
@@ -183,13 +183,23 @@ angular.module('patternfly.filters').directive('pfSimpleFilter', function () {
         return foundFilter !== undefined;
       };
 
+      $scope.enforceSingleSelect = function (filter) {
+        if (filter.type === 'select') {
+          _.remove($scope.config.appliedFilters, {title: filter.title});
+        }
+      };
+
       $scope.addFilter = function (field, value) {
         var newFilter = {
           id: field.id,
           title: field.title,
+          type: field.filterType,
           value: value
         };
         if (!$scope.filterExists(newFilter)) {
+
+          $scope.enforceSingleSelect(newFilter);
```

Comment:
So I would change the order of this code. You want to make the check for if (filter.type === 'select')  here and then if true call enforceSingleSelect. Also should this be an option or do we want this as the default.